package.json

```diff
@@ -13,7 +13,7 @@
   },
   "repository": "https://github.com/notwaldorf/tiny-care-terminal.git",
   "dependencies": {
-    "ansi-art": "^1.2.0",
+    "ansi-art": "1.2.3",
```

Comment:
Please change the `1.2.3` to something forward compatible like `^1.2.3`. Thx!

Reply:
Sure 👍